openfga/v1/errors_ignore.proto

```diff
@@ -57,6 +57,20 @@ enum ErrorCode {
   duplicate_contextual_tuple = 2055;
 }
 
+enum AuthNErrorCode { 
+
+  no_authn_error = 0;
+
+  unauthorized = 3000;
+  missing_bearer_token = 3001;
+	error_parsing_token = 3002;
+	invalid_token = 3003;
+	invalid_claims = 3004;
```

Comment:
Should these be here?

Reply:
Re: these
```

  invalid_claims = 3004;
  invalid_issuer = 3005;
  invalid_audience = 3006;
  invalid_subject = 3007;
```

Reply:
These errors appear, e.g., https://github.com/openfga/openfga/blob/170a6834d057428e3b0d250cae47a01f5a61898f/server/authn/oidc/oidc.go#L76, but it could be generalized to `invalid_cliams` alone. Or perhaps something even more general? `invalid_bearer_token`? `unauthorized`? WDYT?

Reply:
Why are those in there though? 🤔 When would you hit that case?

If they are being used, let's keep them though

Reply:
In the oidc case it validates all these claims against the configured values.